src/main.cpp

```diff
@@ -130,36 +106,25 @@ int main(int argc, char *argv[])
 
 #ifdef LOG_TO_FILE
     QString logFileDir = Settings::getInstance().getSettingsDirPath();
-    logFileStream.reset(new QTextStream);
-    logFileFile.reset(new QFile(logFileDir + "qtox.log"));
+    QString logfile = logFileDir + "qtox.log";
+    logFileFileNO = open(logfile.toLocal8Bit().constData(), O_WRONLY | O_APPEND | O_CREAT);
 
-    // Trim log file if over 1MB
-    if (logFileFile->size() > 1000000) {
+    if (QFileInfo(logfile).size() > 1000000) {
         qDebug() << "Log file over 1MB, rotating...";
 		
         QDir dir (logFileDir);
-		
         // Check if log.1 already exists, and if so, delete it
-        if (dir.remove(logFileDir + "qtox.log.1"))
+        if (dir.remove(logFileDir + "qtox.log.1")) {
             qDebug() << "Removed log successfully";
-        else
+        } else {
             qDebug() << "Unable to remove old log file";
+        }
```

Comment:
No. This too isn't needed: https://github.com/tux3/qTox/wiki/Coding#coding-style